.github/workflows/ci.yml

```diff
@@ -396,6 +396,27 @@ jobs:
                           --cov-report=term --exitfirst -s -vvv --log-cli-level=INFO \
                           ./tests/kubernetes
 
+    - name: HDFS test
```

Comment:
Seems there missing a line `env:`? I think it should be

```
- name: HDFS test
  env:
    CHANGE_MINIKUBE_NONE_USER: ..
    GS_IMAGE: ...
```

Otherwise the CI won't be triggered as there's a syntax error.